bindings/go/src/examples/perftest/perftest.go

```diff
@@ -28,21 +28,21 @@ type PerfTestParams struct {
 	port          uint
 	wakeup        bool
 	ip            string
-	printIter     uint
+	printInterval uint
 	warmUpIter    uint
 }
 
 type PerfTest struct {
 	context              *UcpContext
 	memory               *UcpMemory
 	memParams            *UcpMemAttributes
-	perThreadWorkers     []*UcpWorker
+	worker     	     *UcpWorker
 	listener             *UcpListener
-	eps                  []*UcpEp
-	reverseEps           []*UcpEp
-	numCompletedRequests uint32
+	ep                   *UcpEp
+	numCompletedRequests int32
 	wg                   sync.WaitGroup
-	completionTime       []time.Duration
+	quit		     chan struct{}
+	wake		     []chan struct{}
```

Comment:
```suggestion
	quit                 chan struct{}
	wake                 []chan struct{}
```

Reply:
Spaces are used to align members in other structures. Please use spaces instead of tabs.